praw/endpoints.py

```diff
@@ -2,7 +2,7 @@
 
 # flake8: noqa
 # fmt: off
-API_PATH = {
+API_PATH= {
```

Comment:
👾

Reply:
That should have been detected, and I am going to fix that.